### DISCUSSION TITLE
Editing head models in freesurfer

### DISCUSSION THREAD

**Discussion by Jalyssa112** [simnibs/simnibs#432] [7.Creating head models] [answered]
Hello,

I have been trying to modify head models in Freesurfer to create a new head model to run in SimNIBS. I have tried to follow the tutorial on SimNIBS (https://simnibs.github.io/simnibs/build/html/tutorial/advanced/fix_headmodel.html) to use the voxel edit but when I try to draw on the model, it does not do anything. 

I have included two images for reference, but if anyone has any suggestions I appreciate it!

![IMG_4457](https://github.com/user-attachments/assets/fb5f2c10-f056-4518-a4af-40b7a889b772)
![IMG_4463](https://github.com/user-attachments/assets/5ad072c8-9d1e-4e7d-8507-35ca8a6b6563)

**Comment by oulap** [comment, marked as answer]
This seems to have something to do with data types. As a workaround, you can fire up the simnibs_python interpreter and run these commands:
```

import nibabel as nib
im = nib.load('tissue_labeling_upsampled.nii.gz')
data = im.get_fdata()
im_tmp = nib.Nifti1Image(data,im.affine)
nib.save(im_tmp,'tissue_upsampled_converted.nii.gz')
```
Note that here I'm assuming that you are running these commands in the label_prep folder. 
We'll try to come up with a more permanent fix in a future release.

**Reply by Jalyssa112** [reply]
Yes, I am using label_prep, thank you for the help!!
